### ISSUE TITLE
Helm3: Lint does not catch bad template yamls

### ISSUE THREAD

**Issue by karuppiah7890** [helm/helm#6527]
Output of `helm version`:
```
version.BuildInfo{Version:"v3.0.0-beta.3", GitCommit:"5cb923eecbe80d1ad76399aee234717c11931d9a", GitTreeState:"clean", GoVersion:"go1.12.9"}
```

This is related to the issue #6418 occurring in helm v2. Both have the same root cause.

[chart.zip](https://github.com/helm/helm/files/3666518/chart.zip) . The deployment yaml's `kind` key doesn't have `:` after it

```
$ helm lint chart/
==> Linting chart/
[INFO] Chart.yaml: icon is recommended

1 chart(s) linted, 0 chart(s) failed
```

I'll raise a PR for this too!

**Comment by technosophos** [comment]
I am not sure what the status is here. And is this ONLY a WIndows bug? It's difficult to go from this issue to the linked issue and figure out how much of that issue is supposed to be applied to this issue.

**Comment by karuppiah7890** [comment]
@technosophos yes, this is only a windows bug, based on the file path separator being different in windows

**Comment by karuppiah7890** [comment]
I have closed the PR #6757 . If anyone is interested to take a stab at fixing this issue, please go ahead :)
